### PR TITLE
NOISSUE - Add authorization HTTP API to things service

### PR DESCRIPTION
**Pull request by anovakovic01**
### What does this do?
Adds authorization HTTP API to things service.

### Which issue(s) does this PR fix/relate to?
There is no issue for this.

### List any changes that modify/break current functionality
I've updated existing gRPC env var.

### Have you included tests for your changes?
Yes/

### Did you document any new/modified functionality?
I've updated swagger docs.

### Notes
HTTP API is served on a separate port because this API should be used only internally.

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: things/service.go [hunk 1]
```diff
@@ -403,5 +403,6 @@ func (ts *thingsService) hasThing(chanID, key string) (string, error) {
 		return "", ErrUnauthorizedAccess
 	}
 
+	// fmt.Println(thingID)
```
Comment (dborovcanin) [inline things/service.go]: Please remove commented code.

**Review by drasko — DISMISSED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**